### PR TITLE
feat: add change-side transformer for v6 ui-kit

### PR DESCRIPTION
**Pull request by eyoue**
https://github.com/gravity-ui/uikit-codemod/issues/3
https://github.com/gravity-ui/uikit-codemod/issues/4

### CODE REVIEW

**Comment by eyoue** [comment]
need check it

File: transforms/helpers.ts [hunk 1]
```diff
@@ -3,33 +3,55 @@ import {Collection, JSCodeshift, StringLiteral} from 'jscodeshift';
...
+    root.find(
+        j.JSXElement,
+        childrenName
+            ? {
+                  openingElement: {
+                      name: {
+                          type: 'JSXMemberExpression',
+                          object: {
+                              type: 'JSXIdentifier',
+                              name: foundImport.local?.name || foundImport.imported.name,
```
Comment (amje) [inline transforms/helpers.ts]: Why `childrenName` not used later? Looks like `Button.Something` would also match, but that's an error

**Review by amje — CHANGES_REQUESTED** [review]

File: transforms/6-change-side/index.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+};
+
+export default function (file: FileInfo, api: API, options: Options) {
+    const j = api.jscodeshift;
+    const root = j(file.source);
+
+    if (
+        !Math.max(
+            Number(remapJSXProps('Button.Icon', config, root, j)),
+            Number(remapJSXProps('ButtonIcon', config, root, j)),
```
Comment (amje) [inline transforms/6-change-side/index.ts]: ButtonIcon is not exported from uikit

**Review by amje — CHANGES_REQUESTED** [review]

File: transforms/6-change-side/index.ts (hunk 2 above)
Comment (eyoue) [reply transforms/6-change-side/index.ts]: Then why is it indicated in the issue?
https://github.com/gravity-ui/uikit-codemod/issues/3

File: transforms/helpers.ts (hunk 1 above)
Comment (eyoue) [reply transforms/helpers.ts]: fixed

File: transforms/6-change-side/index.ts (hunk 2 above)
Comment (amje) [reply transforms/6-change-side/index.ts]: By accident. Nevermind, let's keep it.

**Review by amje — APPROVED** [review]

**Pull request merged**